openhtf/plugs/__init__.py

```diff
@@ -258,10 +275,11 @@ def plug(update_kwargs=True, **plugs):
   Raises:
     InvalidPlugError: If a type is provided that is not a subclass of BasePlug.
   """
-  for a_plug in plugs.itervalues():
-    if not issubclass(a_plug, BasePlug):
+  for plug in plugs.itervalues():
```

Comment:
ahh, I was wondering why it was `a_plug` before... I looked everywhere but the function I was in :)